collector/kernel/process_handler.cc

```diff
@@ -18,67 +18,25 @@
 
 #include <collector/agent_log.h>
 #include <common/collected_blob_type.h>
-#include <util/file_ops.h>
 #include <util/log.h>
 #include <util/log_formatters.h>
-#include <util/proc_ops.h>
 #include <util/system_ops.h>
-#include <util/time.h>
 
 #include <generated/flowmill/agent_internal.wire_message.h>
 
 #include <cassert>
 
-constexpr auto PROC_BLOB_SEND_COOLDOWN = 10s;
-
-namespace {
-
-// size in bytes to pre-allocate for the parsing buffer
-// estimated based on maximum size of `/proc/[pid]/{stat|status|io}` files
-constexpr std::size_t PRE_ALLOCATED_PARSING_BUFFER_CAPACITY = 1200;
-
-} // namespace
-
 ProcessHandler::ProcessHandler(
-    ::flowmill::ingest::Writer &writer,
-    ::flowmill::kernel_collector::Index &collector_index,
-    ProbeHandler &probe_handler,
-    ebpf::BPFModule &bpf_module,
-    logging::Logger &log,
-    CpuMemIoSettings const *cpu_mem_io_settings)
-    : writer_(writer),
-      collector_index_(collector_index),
-      log_(log),
-      cpu_mem_io_settings_(cpu_mem_io_settings),
-      memory_page_bytes_(memory_page_size().try_raise().value())
+    ::flowmill::ingest::Writer &writer, ::flowmill::kernel_collector::Index &collector_index, logging::Logger &log)
+    : writer_(writer), collector_index_(collector_index), log_(log), memory_page_bytes_(memory_page_size().try_raise().value())
 {
   LOG::trace_in(
       AgentLogKind::CPU_MEM_IO,
```

Comment:
All remaining uses of AgentLogKind::CPU_MEM_IO should probably be changed to AgentLogKind::PID.